src/TopoMojo.Api/appsettings.conf

```diff
@@ -156,6 +156,10 @@
 # Pod__Sddc__AuthUrl =
 # Pod__Sddc__AuthTokenHeader = csp-auth-token
 
+## these settings currently only apply to Proxmox (not vSphere)
+# Pod__Vlan__ResetDebounceDuration = 1000
+# Pod__Vlan__ResetDebounceMaxDuration = 2000
```

Comment:
I think we should increase these defaults a bit, to 2000 and 5000. In my environment, reloading the networks can take between 4 and 9 seconds, so waiting 2 to potentially avoid that I think makes sense. Depends on how often you expect to have multiple users starting/stopping at the same time.